src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,11 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List get() {

```

Comment:
```suggestion
    public List<Bulldozer> get() {

```

Давай не будем использовать не параметризированный список